docs/content/testing/writing-a-new-test.md

```diff
@@ -0,0 +1,314 @@
+[google-docs-link]: https://docs.google.com/document/d/14_FY-Chu_illhaCCym-pv6P8NHtyN9PLzVBGSuK0ou4/edit (Please place reviews as comments into this document here)
+
+**Warning:** The [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../../README.md#recommended-use).
+
+-----
+
+# Writing a new test
+
+This document covers Unity tests only (that is tests of the Unity project which forms part of the SpatialOS Unity GDK). It walks through creating tests for a made-up class to explain how you can test parts of the Unity GDK. You need to get to know NUnit and the Unity Test Runner before creating tests. See below for more information on these.
+
+## Prerequisites
+
+Please look through and understand the follow documentation and videos before writing a new test:
+* SpatialOS Unity GDK documentation - [Testing overview](./test-overview.md)
```

Comment:
broken link, should be `testing-overview.md`

Reply:
Fixed